### ISSUE TITLE
Flag IX Peering Sessions Without Corresponding PeeringDB Entry

### ISSUE THREAD

**Issue by mngan** [peering-manager/peering-manager#720]
### Peering Manager version

1.7.4

### Feature type

New functionality

### Proposed functionality

Help identify stale IX Peering Sessions by flagging Sessions don't have a corresponding NetIXLan entry. 

### Use case

This flag can be used to filter and possibly clean up sessions on an IX. 

### Database changes

Possibly none, unless you want to cache this state in the session. 

### External dependencies

None

**Comment by gmazoyer** [comment]
This functionality is half there, see [1](https://github.com/peering-manager/peering-manager/blob/e81c424acf99c5c2e060ecc99d64061d7a8a51b0/peering/models/models.py#L895).

The proposed feature could be another version of this function, removing check related to session state, and exposed as a property.

**Comment by mngan** [comment]
I agree that separating this out would allow the operators to define their own logic for what is abandoned/stale. In general though, the combination you have is reasonable.
